src/pages/hardware/guides/mimeProtocol.tsx

```diff
@@ -304,6 +304,6 @@ const pageSections: TableOfContentsItem[] = [
     }
 ];
 
-export default function MimeProtocol() {
+export default function mimeProtocol() {
```

Comment:
components should be Pascal case
```suggestion
export default function MimeProtocol() {
```